custom_components/hubitat/lock.py

```diff
@@ -96,6 +100,22 @@ async def async_unlock(self, **kwargs: Any) -> None:
     async def clear_code(self, position: int) -> None:
         await self.send_command(DeviceCommand.DELETE_CODE, position)
 
+    async def get_codes(self) -> ServiceResponse:
+        response = await self.send_request(DeviceCommand.GET_CODES)
```

Comment:
Since the codes are already in the device state and just not exposed, could they just be retrieved using `self.get_json_attr(DeviceAttribute.LOCK_CODES)` rather than making a new request to the device?

Reply:
That's a good idea.  I'll test that again.  I went down that road briefly, but the codes were seemingly cached and I wasn't getting updates, but later I realized my dev HA instance was misconfigured and wasn't getting notifications from the Hubitat.  It didn't occur to me to circle back to that original idea.  I'll test and update the PR if that works.